src/ape/managers/converters.py

```diff
@@ -70,6 +70,18 @@ def convert(self, value: Any) -> int:
         return to_int(HexBytes(value))
 
 
+class HexListConverter(ConverterAPI):
+    """
+    Convert list of hex values to single concatenated hex value.
```

Comment:
> Convert list of hex values to single concatenated hex value.

Should be `Convert list of hex values to single concatenated HexBytes value` so it is not confused with a hex-str

(not sure why GH not letting use the suggest-code feature on this PR)